doc/getting-started/apl-nuc.rst

```diff
@@ -231,6 +231,20 @@ partition. Follow these steps:
       # clr-boot-manager set-timeout 20
       # clr-boot-manager update
 
+#. Add new user
+
+   .. code-block:: none
+
+      # useradd cl-sos
+      # passwd cl-sos
+      # usermod -G wheel -a cl-sos
+
+#. Enable weston service
+
+   .. code-block:: none
+
+      # systemctl enable weston@cl-sos
```

Comment:
Can you confirm there is no need to reboot anything in between?

Reply:
confirmed , user need to enable westion before reboot, because there are in Native Clear at this moment. otherwise, user  have had into a Back screen 
so people need to get all ready before reboot with ACRN running.